modules/special-rules/pages/jump-packs.adoc

```diff
@@ -1,2 +1,38 @@
 = Jump Packs
 
+These units are equipped with jet-powered devices or other technology, wings, or even powerful long legs that allow them to make long jumps across the battlefield.
+The resultant speed and mobility usually makes for good assault troops.
+
+---
+
+Where your unit has the {jump-packs} ability, apply this change relative to its base unit:
+
+* Create a split Speed value.
```

Comment:
A minor suggestion to cover a gap between creating the split value and applying it to the profile:

```suggestion
* Create and apply a split Speed value.
```

Reply:
Ditto my comment at cavalry.adoc.

Reply:
Copying the revised suggestions here for completeness.

```suggestion
* The unit has a split Speed value, created in this way.
```

Or

```suggestion
* The unit's Speed characteristic has two values.
```

Reply:
Copying the most recent suggestion here so that it can be committed easily.

```suggestion
* Make its Speed value into a split Speed value.
```